tinytodo/policies.cedar

```diff
@@ -1,7 +1,7 @@
 // Policy 0: Any User can create a list and see what lists they own
 permit (
     principal,
-    action in [Action::"CreateList", Action::"GetLists"],
+    action == Action::"CreateList",
```

Comment:
Why did you drop `GetLists`? Just because you are using partial evaluation, I don't see why the situation has changed. The old code authorized both on `GetLists` and individual calls to `GetList`.

Reply:
You may think this was a bad design, but if you change it, we would need to go change all the docs, blog posts, slides, etc. ...

Reply:
Restored.